spec/features/recipes/show_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe "Recipes show page" do
+    it "Shows recipes attributes and list of ingredients" do
+        cookies = Recipe.create!(name: "Cookies", complexity: 2, genre: "Dessert")
+        ingredient_1 = cookies.ingredients.create(name: 'Eggs', cost: 2)
+        ingredient_2 = cookies.ingredients.create(name: 'Butter', cost: 4)
+        ingredient_3 = cookies.ingredients.create(name: 'Chocolate Chips', cost: 5)
+
```

Comment:
In your set up you will probably also need to create a `RecipeIngredient` association.
```ruby
RecipeIngredient.create!(recipe: cookies, ingredient: ingredient_1)
```